### PR TITLE
Add DbContext class to interact with the database

### PR DESCRIPTION
**Pull request by TRybina132**

### CODE REVIEW

File: DataAccess/Context/ClinicContext.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        public DbSet<Address> Addresses { get; set; }
+        public DbSet<Animal> Animals { get; set; }
+        public DbSet<Appointment> Appointments { get; set; }
+        public DbSet<Portfolio> Portfolios { get; set; }
+        public DbSet<Procedure> Procedures { get; set; }
+        public DbSet<Role> Roles { get; set; }
+        public DbSet<Salary> Salaries { get; set; }
+        public DbSet<Specialization> Specializations { get; set; }
+        public DbSet<User> Users { get; set; }
+        public ClinicContext() { }
```
Comment (FedorivRostik) [inline DataAccess/Context/ClinicContext.cs:22]:
```suggestion

```

File: DataAccess/Context/ClinicContext.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        public ClinicContext(DbContextOptions options) : base(options) { }
```
Comment (FedorivRostik) [inline DataAccess/Context/ClinicContext.cs:23]:
```suggestion
        public ClinicContext(DbContextOptions<ClinicContext> options) : base(options) { }
```

**Review by FedorivRostik — CHANGES_REQUESTED** [review]
Add builder.Services.AddDbContext<ClinicContext>(options=>options.UseSqlServer(builder.Configuration.GetConnectionString("DefaultConnection")));

File: Host/Host.csproj [hunk 3]
```diff
@@ -7,11 +7,14 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="6.0.5" />
```
Comment (FedorivRostik) [inline Host/Host.csproj:10]:
```suggestion

```

Comment (FedorivRostik) [reply Host/Host.csproj:10]: We don't need that

Comment (YuraRov) [reply Host/Host.csproj:10]: > We don't need that

Are you sure?

**Review by YuraRov — APPROVED** [review]
Everything seems to be ok :)

**Review by YuraRov — CHANGES_REQUESTED** [review]
Microsoft.EntityFrameworkCore package in Host.csproj is unnecessary

**Review by FedorivRostik — APPROVED** [review]
Alles ist gut

**Review by YuraRov — APPROVED** [review]
All right

**Pull request merged**